python/kfserving/kfserving/api/set_credentials.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2019 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import json
+
+from kubernetes import client
+from ..constants import constants
+
+logger = logging.getLogger(__name__)
+
+
+def create_gcp_credentials(namespace, **kwargs):
```

Comment:
looks we are only passing gcp credential file here, should we make it more explicit ?

Reply:
again should be something like create_gcs_credentials

Reply:
Updated, thanks!